plugins/devlink/meson.build

```diff
@@ -0,0 +1,59 @@
+if host_machine.system() != 'linux'
+  subdir_done()
+endif
+
+if not libmnl.found()
+  subdir_done()
+endif
```

Comment:
we split these out in main to be simpler (see e1f781368b58dff89113db3f52d6dc3b9619394e).  Can you please adjust as well for this plugin?

Reply:
Sure. The code changes to quick :)